examples/ticket-sales/index.txt

```diff
@@ -0,0 +1,5 @@
+Verifying knowledge assertions
+Verifying for generic connector
+  Verifying when ALL participants are honest
+  Verifying when NO participants are honest
+Checked 27 theorems; No failures!
```

Comment:
I think you copy and pasted this file rather than using the gen-test-output script